front/types/Video.ts

```diff
@@ -0,0 +1,12 @@
+type sizes = '144' | '240' | '480' | '720' | '1080';
+
+export interface Video {
+  description: string;
+  id: string;
+  is_ready: boolean;
```

Comment:
Should it be a flag or should we have statuses? For example if transcoding fails in AWS?

Reply:
Good idea. I put `string` for now and will update with the actual literals once we implement them.